cul.js

```diff
@@ -136,7 +141,8 @@ var Cul = function (options) {
 
     this.write = function send(data, callback) {
         //console.log('->', data)
-        serialPort.write(data + '\r\n', callback);
+        serialPort.write(data + '\r\n');
+        serialPort.drain(callback);
```

Comment:
Shouldn't drain be called inside the serialPort.write callback like in the example here https://github.com/EmergingTechnologyAdvisors/node-serialport#module_serialport--SerialPort+drain ?